neuro_san/service/main_loop/server_main_loop.py

```diff
@@ -142,7 +142,7 @@ def parse_args(self):
         self.service_openapi_spec_file = args.openapi_service_spec_path
         self.manifest_update_period_seconds = args.manifest_update_period_seconds
         if self.manifest_update_period_seconds <= 0:
-            # Periodic manifest updater is disabled:
+            # StorageWatcher is disabled:
             self.server_status.updater.set_requested(False)
```

Comment:
@andreidenissov-cog : Should I change server_status.updater to be server_status.watcher?